contentacms/contenta_jsonapi
# Remove invalid recipe from recipes_magazin

When I installed contentacms using the install script, I did not receive any default content. After some troubleshooting, I found that there was an invalid recipe in the recipes_magazin module, which was causing the recipes_magazin__recipes migration to fail. This PR removes the invalid recipe.

### Test Instructions

If you build contentacms from 8.x-1.x you should not receive any default content. If you build from the 8.x-fix-recipes branch, default content should work.
